gltf/gltfpack.cpp

```diff
@@ -896,7 +958,7 @@ int gltfpack(const char* input, const char* output, const char* report, Settings
 			return 4;
 		}
 	}
-	else if (oext && (strcmp(oext, ".glb") == 0 || strcmp(oext, ".GLB") == 0))
+	else if (oext && (strcmp(oext, ".vrm") == 0 || strcmp(oext, ".VRM") == 0 || strcmp(oext, ".glb") == 0 || strcmp(oext, ".GLB") == 0))
```

Comment:
Should this also be added to the condition above that sets `texture_embed`? Unclear what the de-facto rules are for VRM files, e.g. whether they are supposed to keep textures separately or not.